### PR TITLE
cilium: convert to common.mk

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/153#" title="Author self-approved">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by abhay-krishna** [comment]
@jaxesn: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
cilium-tooling-presubmit | 9225ebc35a3c5e5f01d6676dc813da3e6b4a858e | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/153/cilium-tooling-presubmit/1458448166276829184) | `/test cilium-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere-build-tooling&pr=153). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:jaxesn).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: projects/cilium/cilium/Makefile
Comment (jaxesn) [inline projects/cilium/cilium/Makefile:14]: In a future PR I have a change coming that we can use to improve this nonsense.

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**
